### ISSUE TITLE
测试jhmdb数据集

### ISSUE THREAD

**Issue by lonf69** [bo-miao/SgMg#17]
你好 请问针对于jhmdb数据集的测试 我按照代码里的提示用的是在a2d数据集上微调出来的模型直接给jhmdb数据集做的测试 但是效果显示很差 请问这个是什么问题

**Comment by bo-miao** [comment]
I cannot provide suggestions without any details. It is recommended not to modify the code initially to ensure correctness.
You should first pretrain the model on Ref-COCOs, then finetune it on A2D.

**Comment by lonf69** [comment]
我看代码中注释的意思是直接用A2D微调好的模型直接给jhmdb数据集做测试 但是我这样尝试之后发现效果不好 各项指标都是0

**Comment by bo-miao** [comment]
Correct, same checkpoint is used for both A2D and JHMDB. If the performance is correct on A2D, you may want to check the checkpoint loading or data organization when evaluating on JHMDB.

**Comment by lonf69** [comment]
python inference_ytvos.py --with_box_refine --binary --freeze_text_encoder \
  --eval \
  --ngpu=${GPUS_PER_NODE} \
  --output_dir=${OUTPUT_DIR} \
  --resume=${CHECKPOINT} \
  --backbone=${BACKBONE} \
  --backbone_pretrained=${BACKBONE_PRETRAINED} \
  --visualize \
  --amp \ 
您好 运行ytvos测试代码后只有分割的可视化掩码 没有J&F等指标 这个是要添加什么参数嘛还是如何操作

**Comment by lonf69** [comment]
============================================>
JHMDB-Sentences are directly evaluated using the checkpoint trained on A2D-Sentences
Load checkpoint weights from /home/wr506/zlf/SgMg/checkpoints/results/SgMg_video_swin_t_p4w7_finetune_a2d/checkpoint.pth ...
============================================>

 **** Begin to evaluating jhmdb. **** 

Test:  [  0/696]  eta: 0:23:47    time: 2.0505  data: 1.0164  max mem: 2126
Test:  [100/696]  eta: 0:01:37    time: 0.1443  data: 0.0036  max mem: 2156
Test:  [200/696]  eta: 0:01:15    time: 0.1426  data: 0.0035  max mem: 2156
Test:  [300/696]  eta: 0:00:59    time: 0.1426  data: 0.0035  max mem: 2156
Test:  [400/696]  eta: 0:00:43    time: 0.1433  data: 0.0034  max mem: 2156
Test:  [500/696]  eta: 0:00:28    time: 0.1447  data: 0.0034  max mem: 2156
Test:  [600/696]  eta: 0:00:14    time: 0.1448  data: 0.0033  max mem: 2156
Test:  [695/696]  eta: 0:00:00    time: 0.1418  data: 0.0033  max mem: 2156
Test: Total time: 0:01:42 (0.1466 s / it)
loading annotations into memory...
Done (t=0.04s)
creating index...
index created!
Loading and preparing results...
DONE (t=9.79s)
creating index...
index created!
Running per image evaluation...
Evaluate annotation type *segm*
DONE (t=10.82s).
Accumulating evaluation results...
DONE (t=0.48s).
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.000
evaluating mask precision@k & iou metrics...
100%|█████████████████████████████████████| 2784/2784 [00:01<00:00, 1490.23it/s]
{'mAP 0.5:0.95': 0.0, 'AP 0.5': 0.0, 'AP 0.75': 0.0, 'AP 0.5:0.95 S': 0.0, 'AP 0.5:0.95 M': 0.0, 'AP 0.5:0.95 L': 0.0, 'P@0.5': 0.0, 'P@0.6': 0.0, 'P@0.7': 0.0, 'P@0.8': 0.0, 'P@0.9': 0.0, 'overall_iou': 0.1039332190259047, 'mean_iou': 0.09492282715630075}
  这是评估JHMDB数据集的结果 请问如何检查检查点的加载或数据组织

**Comment by bo-miao** [comment]
Please review the code before making any changes. You are executing the wrong file. The evaluation script should look like this:

CUDA_VISIBLE_DEVICES=${GPUS} OMP_NUM_THREADS=${CPUS_PER_TASK} torchrun --master_port ${PORT}  --nproc_per_node=${GPUS_PER_NODE} main.py \
  --eval \
  --dataset_file jhmdb \
  --with_box_refine --freeze_text_encoder \
  --exp_name=${EXP_NAME} \
  --output_dir=${OUTPUT_DIR} \
  --backbone=${BACKBONE} \
  --backbone_pretrained=${BACKBONE_PRETRAINED} \
  --resume ${CHECKPOINT} \
  --amp \
